Model/Maker/UiComponentGrid.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace Ctasca\MageBundle\Model\Maker;
+
+use Ctasca\MageBundle\Api\MakerUiComponentInterface;
+use Ctasca\MageBundle\Api\MakerUiComponentXmlInterface;
+use Ctasca\MageBundle\Console\Question\Prompt\Validator as QuestionValidator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UiComponentGrid extends AbstractMaker implements MakerUiComponentXmlInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function make(InputInterface $input, OutputInterface $output): void
+    {
+        $question = $this->makeModuleNameQuestion();
+        $moduleName = $this->questionHelper->ask($input, $output, $question);
+        try {
+            /** @var \Ctasca\MageBundle\Model\Template\Locator $templateLocator */
+            list($templateLocator,)  = $this->locateTemplateDirectory(self::UI_COMPONENT_TEMPLATES_DIR);
+            $question = $this->makeGridNamespaceQuestion(
+                $templateLocator,
+                'Please type the namespace for grid Ui component'
+            );
+            $namespace = $this->questionHelper->ask($input, $output, $question);
```

Comment:
Maybe rename $namespace to $xmlFilename to keep it consistent with question made?

Reply:
oltre a essere il nome del file, quel parametro è proprio assegnato a una delle proprietà che ha nome namespace all'interno dell'xml. Lasciamo così? Al limite nella descrizione, potrei specificare che il nome assegnato sarà anche il prefisso del file, dato che la parte finale è statica, cioè "_grid.xml"